### PR TITLE
Jetty 12.0.x osgi

### PR DESCRIPTION
**Pull request by janbartel**
Make osgi work with jetty-12 environments.

### CODE REVIEW

**Comment by joakime** [comment]
Ay! it's green!

Can we merge this now?

**Comment by janbartel** [comment]
Oh please God  can we????!! Somebody review this.

**Comment by joakime** [comment]
@olamy please make this review a priority.

File: jetty-ee10/jetty-ee10-osgi/test-jetty-ee10-osgi/pom.xml [hunk 1]
```diff
@@ -17,9 +17,8 @@
     <pax.url.version>2.6.2</pax.url.version>
     <swissbox.version>1.8.3</swissbox.version>
     <tinybundles.version>3.0.0</tinybundles.version>
-    <spifly.version>1.3.4</spifly.version>
-    <!-- TODO -->
-    <skipTests>true</skipTests>
+    <spifly.version>1.3.6</spifly.version>
```
Comment (olamy) [inline jetty-ee10/jetty-ee10-osgi/test-jetty-ee10-osgi/pom.xml:20]: this version is repeated so many times maybe a single place would be better?

File: jetty-ee10/jetty-ee10-osgi/pom.xml [hunk 2]
```diff
@@ -117,11 +106,36 @@
...
         <version>${osgi-version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.osgi</groupId>
+        <artifactId>org.osgi.service.cm</artifactId>
+        <version>${osgi-service-cm-version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.osgi</groupId>
+        <artifactId>org.osgi.service.component</artifactId>
```
Comment (olamy) [inline jetty-ee10/jetty-ee10-osgi/pom.xml:116]: this and some other repeated would be better in dptMngt in top pom and added here without version (to have a single place with the version and dptMngt is a better way to handle transitivity and possible upper bound issues)

**Review by olamy — APPROVED** [review]
Sounds good (I have added some comments which can/should be fixed but maybe later?)

File: jetty-ee10/jetty-ee10-osgi/pom.xml (hunk 2 above)
Comment (joakime) [reply jetty-ee10/jetty-ee10-osgi/pom.xml:116]: Lets tackle this as a clean PR in a followup PR

**Pull request merged**